src/util/std_expr.h

```diff
@@ -1731,6 +1731,106 @@ inline void validate_expr(const complex_exprt &value)
 }
 
 
+/// \brief Real part
```

Comment:
part ... of what? (I'd say "Real part of the expression describing a complex number.")